build_tools/build_release.py

```diff
@@ -117,7 +117,7 @@ def _download_iree_binaries_for_platform_args(
                 # across projects and new stable versions of the IREE packages
                 # haven't yet been pushed.
                 "-r",
-                REPO_ROOT / "iree-requirements-ci.txt",
+                REPO_ROOT / "requirements-iree-pinned.txt",
```

Comment:
Without looking to deep into the script, I am wondering why it is needed to install those requirements when building a release. Anyway the change is fine.

Reply:
The downloading is mostly leftover from a previous iteration of the release process where IREE wheels would be pushed _from iree-turbine_:

1. From iree-turbine, run this script
2. The script downloads candidate wheels from IREE, builds iree-turbine, and runs iree-turbine tests
3. Push all IREE and iree-turbine wheels to PyPI _from iree-turbine_

Lately I've been doing this instead:

1. Choose a release candidate in IREE
2. Switch to iree-turbine then set `iree-requirements-ci.txt` / `requirements-iree-pinned.txt` to that release candidate and send a PR that runs CI tests then merge it
3. Switch back to IREE and publish the IREE wheels
4. Switch back to iree-turbine, build wheels, and publish

Reply:
Thanks for enlightening me :)